rom/dev/src/start.S

```diff
@@ -104,14 +104,21 @@ _start:
     andi t1, t1, 0x3
     bne t1, x0, post_ecc_init
 
+    //
     // Cold Boot
+    //
+
+    // Zero ICCM
     la a0, ICCM_ORG         // dest
     la a1, ICCM_SIZE        // len 
     call _zero_mem256
 
-    la a0, DCCM_ORG         // dest
-    la a1, DCCM_SIZE        // len 
-    call _zero_mem256
+    // Zero out post-CFI Entropy region
+    // CFI region is used as an additional entropy source for CFI RNG,
+    // hence it is not cleared.
+    la a0, DCCM_POST_CFI_ENTROPY_ORG     // dest
+    la a1, DCCM_POST_CFI_ENTROPY_SIZE    // len 
+    call _zero_mem32
```

Comment:
Can we reserve the first 8 words and use zero_mem256? The unrolled loop is WAY faster.